webapp/packages/core-app/src/useClientActivity.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { useEffect } from 'react';
+
+import { ClientActivityService } from '@cloudbeaver/core-activity';
+import { useService } from '@cloudbeaver/core-di';
+
+export function useClientActivity() {
+  const clientActivityService = useService(ClientActivityService);
+
+  function updateActivity() {
```

Comment:
this one executes a lot. I suppose we can add here a throttle with 100ms throttle. what do you think, guys?

Reply:
yep it's a good idea you can use `throttle` from `core-utils`